### ISSUE TITLE
Requested Feedback 11/26

### ISSUE THREAD

**Issue by Enspired** [emilykmurray/GDMA-2662-Assignment-3#2]
- [ ] There appears to be an extra HTML page on github 'whatwedo', manually delete any unused HTML pages from Github
- [x] Consider title that full describe the type of page 'Home | Bedford Foundation' or 'Bedford Foundation' https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/index.html#L12 
- [x] Alt text should be the same content as what's displayed in browser. Update for all pages https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/index.html#L21 

Index 
- [x] Incorrect link https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/index.html#L34 
- [x] Extra div can be removed https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/index.html#L39 
- [ ] .col-center only works when it's applied to a .row. Either include another div.row.col-center around the .col-lg-7 or remove .col-center class https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/index.html#L42 
- [x] Extra div can be removed https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/index.html#L50 https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/index.html#L62

About
- [x] Appears to be an extra div closing tag. Validating should catch these errors https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/about.html#L32 
- [x] Extra div can be removed. .wrapper.row can be moved to main element https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/about.html#L37 
- [x] Double check that these are linking to the correct locations on the page https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/about.html#L42 
- [x] This is not a section, a container is not needed around the paragraph https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/about.html#L50 
- [x] Closing content for the blockquote should be inside a fotoer https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/about.html#L60 
- [x] The header should be outside the row so that the two list align at the top in relation to each other  https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/about.html#L66 
- [ ] Paragraph tags aren't needed inside the li's but it's also not incorrect to have them https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/about.html#L70 

Take Action 
- [ ]  'justify-content' that .col-center is using needs to be on a .row to work. Otherwise use margin: 0 auto; to center the container https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/takeaction.html#L36 
- [x] These links can just be pound signs '#' so that they don't refresh the page https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/takeaction.html#L46 

CSS 
- [x] Be sure to define the base / root font-size on the HTML element 
- [x] ID selectors should be grouped under Components https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/css/main.css#L60 
- [x] .btn is a utility class https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/css/main.css#L70 
- [ ] border-radius is a CSS3 property and can have vendor prefixes https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/css/main.css#L312 
- [ ] Group repeating styles within Components https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/css/main.css#L336 
- [x] Simplify selectors, this should be able to target just the 'h1' without negative impact to the home page https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/css/main.css#L360 
- [x] Outside of the media query you're using margin-top but inside the media query you're using padding-top. The values are the same, why the change? https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/css/main.css#L393 
- [x] Styles that aren't changing between display sizes do not need to be repeated inside the media query https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/css/main.css#L410 
- [x] Be sure to use margin to create space between elements. The only time text should have padding is when it has a background color https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/css/main.css#L434 
- [ ] Do not divide styles based on the page it's targeting, but on the component. All heros should be grouped together under Components > Hero https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/css/main.css#L431 
- [x] The nav shouldn't need margin: 0 auto; if text align is being used to center the text https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/css/main.css#L505 
- [x] If you're going to be use the #hero IDs to target the nav for some instances, use it for all instances to reduce the amount of CSS you need to override when targeting the #postion-sticky nav https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/css/main.css#L509 
- [ ] Consider contextual selectors before using IDs. The #position-sticky nav is the only nav inside the main element, all other navs are inside the header element. Simplify selectors, to target all navs inside the header, you do not need to utilize the ID for each #hero, use a contextual selector
- [x] All hero styles should be grouped under Components > Hero, and reduce repeating declarations (group repeating styles) https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/b61aa85567b4b50c2873b3da7df65cfbe26830d9/css/main.css#L655

**Comment by Enspired** [comment]
- [x] The image does not say 'logo' in browser so it should not say 'logo in the alt text https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/8774da2ba0864df05943606434a5f53e06347cfb/index.html#L21 
- [ ] If you have three paragraphs on three different pages that all need to be styled the same, consider a class rather than three different IDs https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/8774da2ba0864df05943606434a5f53e06347cfb/takeaction.html#L36 


Index 
- [x] .col- class is not needed here, the text just needs to be aligned to the center https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/8774da2ba0864df05943606434a5f53e06347cfb/index.html#L81 

About 
- [x] Extra closing div tag, be sure to validate all pages https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/8774da2ba0864df05943606434a5f53e06347cfb/about.html#L32 
- [x] Extra div can be removed, place the .wrapper.row on the main element https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/8774da2ba0864df05943606434a5f53e06347cfb/about.html#L37 
- [ ] If you move the paragraph inside the nav element, you could get rid of the div and position sticky  ID. https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/8774da2ba0864df05943606434a5f53e06347cfb/about.html#L39 

CSS
- [x] Should be under Generic https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/8774da2ba0864df05943606434a5f53e06347cfb/css/main.css#L45 
- [x] Be sure grouped selectors are above the subcategory 'typography' 
- [x] Blockquote can have its own category under Elements 
- [ ] Utilize font shorthand when applying family and size 
- [ ] Group repeating styles under Elements such as color, font family, text align 
- [x] Simplify selectors, this can just be '.btn:hover' https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/8774da2ba0864df05943606434a5f53e06347cfb/css/main.css#L153 
- [x] All .btn selectors should be under Utility since this is a utility class 
- [ ] Styles that aren't changing across display sizes do not need to be repeated inside the media query https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/8774da2ba0864df05943606434a5f53e06347cfb/css/main.css#L379 
- [x] Remove repeating property within one declaration block https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/8774da2ba0864df05943606434a5f53e06347cfb/css/main.css#L448 
- [x] If you have navigation under Components, no navigation styles should be under Elements 
- [x] Simplify selectors, all of these can be targeted with 'header nav' https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/8774da2ba0864df05943606434a5f53e06347cfb/css/main.css#L540 
- [x] This is all targeting another nav, group by Components, not by page https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/8774da2ba0864df05943606434a5f53e06347cfb/css/main.css#L613 
- [x] Remove unused styles https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/8774da2ba0864df05943606434a5f53e06347cfb/css/main.css#L646 
- [x] This padding is causing the image to exceed the width of its parent. The width by default is going to be 100%. Padding is only needed for a height, or a height can be set since there is no content in the div https://github.com/emilykmurray/GDMA-2662-Assignment-3/blob/8774da2ba0864df05943606434a5f53e06347cfb/css/main.css#L671 
- [x] All hero styles should be grouped together regardless of what CSS is being applied
